BUILD_BLOSC.bat

```diff
@@ -3,26 +3,57 @@ setlocal EnableDelayedExpansion
 :: Libraries from local folder
 @SETLOCAL
 @ECHO off
-call "%VS140COMNTOOLS%..\..\VC\bin\vcvars32.bat"
+call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x64
 
 set OUT_DIR=.
 
+:: Remove existing Dll's
+if exist "%OUT_DIR%\blosc\Release" rmdir /s /q "%OUT_DIR%\blosc\Release"
+if errorlevel 1 exit /b %errorlevel%
+
 ::Assume this script lives in root of blosc repo
 set "lastCompileAttempt=Blosc"
 ::create visual studio project with cmake
 @ECHO ON
 cmake -DCMAKE_INSTALL_PREFIX="%OUT_DIR%" --build .
 @ECHO OFF
-if !errorlevel! neq 0 exit /b !errorlevel!
+if errorlevel 1 exit /b %errorlevel%
 
 ::build visual studio
-@ECHO ON
 msbuild blosc.sln /p:Configuration=Release /p:Platform=Win32 
-@ECHO OFF
-if !errorlevel! neq 0 exit /b !errorlevel!
+::@ECHO OFF
+if errorlevel 1 exit /b %errorlevel%
 popd
 
 
+::remove all files used for building

```

Comment:
If I am not mistaken the PRE_  just gathers the data and the POST_ cleans up the extra files and folders that were built. In this case, the script lives in the repo and just builds the necessary artifacts so it would be a BUILD script.
I can move the cleaning into the ConsolidateArtifacts or a separate CLEAN_ script if that's preferred. I think not bloating the folder with extra files is a good thing especially if we won't build more than once.
By using the git commands I would have to remove all files and folders from the repo even the artifacts folder as it is not part of the repository. In this case, a separate CLEAN script would work best